mxcube3/routes/Queue.py

```diff
@@ -535,6 +535,7 @@ def add_centring(id):
         409 something bad happened. Plus:
        data ={ "CentringId": newId}
     '''
+    params = request.get_json()
```

Comment:
The client will never be able to add a centring ~directly, i.e., it will be done through a datacollection. You are right about the missing ```params```, but perhaps we can remove completely ```add_centring``` method?